spec/jobs/remove_from_algolia_index_job_spec.rb

```diff
@@ -0,0 +1,6 @@
+require "rails_helper"
+require "jobs/shared_examples/enqueues_job"
+
+RSpec.describe RemoveFromAlgoliaIndexJob, type: :job do
+  include_examples "#enqueues_job", "remove_from_algolia_index", "users-456"
```

Comment:
I suggest passing arguments that are similar to existing ones to the job, like "searchables_#{Rails.env}", for clarity.